pkg/ctl/cmdutils/cmdutils.go

```diff
@@ -98,13 +99,21 @@ func AddCommonFlagsForAWS(group *NamedFlagSetGroup, p *api.ProviderConfig, cfnRo
 		if err := fs.MarkHidden("aws-api-timeout"); err != nil {
 			logger.Debug("ignoring error %q", err.Error())
 		}
-		fs.DurationVar(&p.WaitTimeout, "timeout", api.DefaultWaitTimeout, "max wait time in any polling operations")
+		if timeout {
```

Comment:
Boolean parameters are very obscure and don't help readability. If what we want to do is tweak the timeout message then maybe that's the parameter that should be passed to this function. what do you think?

Reply:
Btw, why not just say "maximum waiting time for any long-running operations"?

Reply:
I was hesitant about this `bool` as I don't like it either.
However, I didn't want to have too many arguments on `AddCommonFlagsForAWS` hence I went for it nevertheless, for now.

What I'd rather have is:
- move `--timeout` out of the "AWS" arguments section completely, to have it in the "General" sections for all calls, and then
- have a more generic description like you are suggesting.

Reply:
I've done what I've described above, PTAL.